### ISSUE TITLE
command 'vscode-docker.images.selectGroupBy' already exists

### ISSUE THREAD

**Issue by tvilot** [microsoft/vscode-docker#1008]
Repro steps:

1.
2.

Action: docker.activate
Error type: Error
Error Message: command 'vscode-docker.images.selectGroupBy' already exists

Version: 0.6.3
OS: linux
Product: Visual Studio Code
Product Version: 1.35.1
Language: en

<details>
<summary>Call Stack</summary>

```
d.registerCommand extensionHostProcess.js:497:447
Object.registerCommand extensionHostProcess.js:716:422
Object.module.exports.t.registerCommand extension.bundle.js:211:110531
ue extension.bundle.js:157:609314
Object.<anonymous> extension.bundle.js:157:611927
Object.<anonymous> extension.bundle.js:1:394616
module.exports.n extension.bundle.js:1:392954
module.exports.n extension.bundle.js:1:392731
Object.module.exports.t.callWithTelemetryAndErrorHandling extension.bundle.js:1:394533
Object.module.exports.t.activateInternal extension.bundle.js:157:610822
```

</details>

**Comment by EricJizbaMSFT** [comment]
@tvilot we just changed publisher from "PeterJausovec" to "ms-azuretools". It's possible your machine still has an extension under the old publisher. Can you try reloading VS Code?

If that doesn't work - you could also try manually removing the folder like "PeterJausovec.vscode-docker" from your extensions folder:
* Windows `%USERPROFILE%\.vscode\extensions`
* macOS `~/.vscode/extensions`
* Linux `~/.vscode/extensions`

**Comment by rrmistry** [comment]
@EricJizbaMSFT , a clean uninstall -> reload VS Code -> install of `ms-azuretools.vscode-docker` seems to have resolved the issue. Thank you!

**Comment by arush0** [comment]
I followed @rrmistry steps but I still have issues with the extension. vscode recommends the extension even though I have it installed.
<img width="437" alt="image" src="https://user-images.githubusercontent.com/32102766/59767582-9aa4e100-9270-11e9-8d32-d919469d60d4.png">

**Comment by rrmistry** [comment]
Hello @arush0 , I was not precise in my comment. I had uninstalled the docker extension and not VS Code. As far as I understand, extensions do not get uninstalled when you uninstall VS code.

My apologies for the unintentional misdirection.

**Comment by arush0** [comment]
@rrmistry No, you were clear enough. I did exactly that. I uninstalled the docker extension and I verified the extension was gone under my `~/.vscode` directory. Even after reloading and re-installing I still see the issue.

**Comment by fiveisprime** [comment]
@arush0 We have an update to VS Code core coming to correct the extension recommendation. Sorry about that!

Right now, the recommendation should fail to find and install the old version of the extension - that's what I'm currently seeing.

**Comment by fiveisprime** [comment]
The recommendation updates will be in Insiders either today or tomorrow and will ship in the next stable release (early July).

Again, so sorry for the issues.

**Comment by arush0** [comment]
But why would that prevent the new extension from working? I don't care so much about the pop up, but the extension isn't working, there's no syntax highlighting, no formatting.

**Comment by IEvangelist** [comment]
I'm experiencing the same issue and I have a clean install of vscode and the following extensions:

```console
code --install-extension Acrolinx.vscode-sidebar
code --install-extension akamud.vscode-theme-onedark
code --install-extension DavidAnson.vscode-markdownlint       
code --install-extension docsmsft.docs-article-templates      
code --install-extension docsmsft.docs-authoring-pack
code --install-extension docsmsft.docs-markdown
code --install-extension docsmsft.docs-preview
code --install-extension docsmsft.docs-yaml
code --install-extension Equinusocio.vsc-material-theme       
code --install-extension ms-azuretools.vscode-docker
code --install-extension ms-vscode.azurecli
code --install-extension ms-vsliveshare.vsliveshare
code --install-extension PKief.material-icon-theme
code --install-extension redhat.vscode-yaml
code --install-extension streetsidesoftware.code-spell-checker
```

**Comment by IEvangelist** [comment]
> @tvilot we just changed the publisher from "PeterJausovec" to "ms-azuretools". It's possible your machine still has an extension under the old publisher. Can you try reloading VS Code?
> As a manual workaround, you can go to your "Extensions" page and uninstall v0.6.2 of the extension with "PeterJausovec" as the publisher:

This doesn't seem to be the issue. I am experiencing it from a clean install.

**Comment by StephenWeatherford** [comment]
@arush0 @IEvangelist @tvilot This should have been fixed with the 0.6.4 update last week.  If that's not the case, could you please let us know?  Thanks!

Current known status:
The most recent update is 0,.6.4 (with the new publisher name)
Vscode still recommends the 0.6.2 extension with the old publisher name (will be fixed with next vscode update)
It should be okay to have both the old and new publisher versions installed (0.6.2 and 0.6.4), everything should still work as expected (once vscode is fixed, we'll put in a patch to recommend uninstalling the old one)

**Comment by IEvangelist** [comment]
> @arush0 @IEvangelist @tvilot This should have been fixed with the 0.6.4 update last week.  If that's not the case, could you please let us know?  Thanks!
> Current known status:
> The most recent update is 0,.6.4 (with the new publisher name)
> Vscode still recommends the 0.6.2 extension with the old publisher name (will be fixed with next vscode update - see #1075)
> It should be okay to have both the old and new publisher versions installed (0.6.2 and 0.6.4), everything should still work as expected (once vscode is fixed, we'll put in a patch to recommend uninstalling the old one)

Great, thank you @StephenWeatherford ... I have been using VS Code without experiencing the issue since I've updated - seems to have been fixed. Thanks again
